tests/base/parser/bad/gold/stmts_t-match-spid-with-bar-cid-cid-with.scilla.gold

```diff
@@ -1,11 +1,9 @@
 {
   "errors": [
     {
-      "error_message":
-        "Invalid type or pattern. If this is a type, then this is likely due to an illegal map value type. If this is a pattern-match, then it is likely due to a misplaced or missing double arrow ('=>').\n",
+      "error_message": "In this match expression, following the pattern the parser expects an arrow (e.g. '=>').\n",
```

Comment:
Reviewed this test case - I think this error message is an improvement. The only potential change is we might want to remove the `If this is a pattern-match...` sentence from the other error, because I believe we no longer hit it erroneously.

Reply:
I like that it is an improvement and we can consider copying it across but for starters, I would like there to be no changes here since, well, there shouldn't be.

Reply:
As per private message, I think we understand this now. Resolving.